src/app/components/modules/Header.jsx

```diff
@@ -82,12 +82,12 @@ class Header extends React.Component {
         const current_account_name = this.props.current_account_name;
         let home_account = false;
         let page_title = route.page;
-
         let sort_order = '';
         let topic = '';
         let user_name = null;
         let page_name = null;
         this.state.subheader_hidden = false;
+        this.context.intl = this.context.intl || { locale: 'en' };
```

Comment:
It's a redundant initialization though required to pass the tests. See https://gist.github.com/mirague/c05f4da0d781a9b339b501f1d5d33c37/